chat-widget/src/components/livechatwidget/common/createAdapter.ts

```diff
@@ -27,8 +27,9 @@ export const createAdapter = async (chatSDK: any) => {
     };
     let adapter = await chatSDK.createChatAdapter(chatAdapterOptionalParams);
     //so far, there is no need to convert to the shim adapter when using visual tests
-    if(chatSDK.isMockModeOn !== true){
+    if (chatSDK.isMockModeOn !== true) {
         adapter = new ChatAdapterShim(adapter);
+        return adapter.chatAdapter;
     }
     return adapter;
```

Comment:
I recommend to `return adapter.chatAdapter` then pass an optional param to pass  subscribers:

`createAdapter(chatSDK, subscribers)`

then `createAdapter` would add all the subscribers

Reply:
I can add it in my next PR,